lib/defaults.json

```diff
@@ -15,6 +15,7 @@
   ["import", "export"],
   ["join", "split"],
   ["JSON.parse", "JSON.stringify"],
+  ["open", "close"],
```

Comment:
Not sure if this one is needed since for me it's a personal config.
What about `["opened", "closed"],`? (my english is bad so I might say nonsense)